Makefile

```diff
@@ -27,21 +29,13 @@ dist: ## generate distribution package
 upload: ## upload distribution package to PyPI
 	python3 -m twine upload dist/*
 
-.PHONY: upgrade-dist-tools
-upgrade-dist-tools: ## upgrade packages necessary for testing, building, packaging and uploading to PyPI
-	python3 -m pip install --upgrade pip setuptools wheel twine pytest mkdocs
-
 .PHONY: docs-build
 docs-build: ## build docs into the local 'site' folder
 	mkdocs build
 
 .PHONY: docs-serve
 docs-serve: ## serve docs for local viewing
-	mkdocs serve
-
-.PHONY: docs-publish
-docs-publish: ## build and publish docs to GitHub Pages
-	mkdocs gh-deploy
+	mkdocs serve --dev-addr=0.0.0.0:7979
```

Comment:
- `docs-publish` is not used anymore, because the docs are only on docs.dasch.swiss.
- for `mkdocs serve`, I need another port, because the standard port is occupied on my machine.